FirebaseMLModelDownloader/Tests/Integration/ModelDownloaderIntegrationTests.swift

```diff
@@ -176,4 +182,167 @@ final class ModelDownloaderIntegrationTests: XCTestCase {
     waitForExpectations(timeout: 5, handler: nil)
     XCTAssertEqual(modelDownloadManager.downloadStatus, .completed)
   }
+
+  func testGetModel() {
+    let testApp = FirebaseApp.app()!
```

Comment:
I can't remember if we've done it elsewhere already - are you able to use an internal initializer for `FirebaseApp` to instantiate it manually instead of using the shared singleton?

Reply:
I haven't used this before, but added a TODO for a future PR.